game-7daystodie/index.tsx

```diff
@@ -133,7 +133,7 @@ async function prepareForModding(context: types.IExtensionContext,
     }
     await fs.ensureDirWritableAsync(path.join(directory, 'Mods'));
     const launcher = DEFAULT_LAUNCHER_SETTINGS;
-    launcher.DefaultRunConfig.AdditionalParameters = `-UserDataFolder=${directory}`;
+    launcher.DefaultRunConfig.AdditionalParameters = `-UserDataFolder=\"${directory}"`;
```

Comment:
The `\` character is not required as the `JSON.stringify` function will escape double quotations automatically where required.